app/core/Engine/messengers/multichain-account-service-messenger/multichain-account-service-messenger.ts

```
@@ -44,6 +44,7 @@ export function getMultichainAccountServiceMessenger(
       'KeyringController:getKeyringsByType',
       'NetworkController:getNetworkClientById',
       'NetworkController:findNetworkClientIdByChainId',
+      'ErrorReportingService:captureException',
```

This is the main breaking change for `@metamask/multichain-account-service`, some errors were not reported back to Sentry, making some error cases hard to track and diagnose